eng/jacoco-test-coverage/pom.xml

```diff
@@ -551,7 +551,7 @@
 	  <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-digitaltwins-core</artifactId>
-      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-digitaltwins-core;current} -->
+      <version>1.1.1</version> <!-- {x-version-update;com.azure:azure-digitaltwins-core;current} -->
```

Comment:
Apparently there is some script to run to update the version number, in the Java repo. Did you use that?

Reply:
I did not, this is usually updated after we do a release automatically. But the release did not go through all the way (an issue with the pipeline that we are working with Azure SDK team to resolve). So I manually did the update. I will ask about the script that does the version bump and run that in the future.

Reply:
Apparently us not running the script caused a problem a while back, so when someone else ran it for their SDK it found a file we missed and updated it too. That caused some consternation. I may have misunderstood it. Please just check with a Java contact to ask what the proper way is.